src/routes/[...404].tsx

```diff
@@ -0,0 +1,8 @@
+import { useLocation } from "@solidjs/router";
+import { getLocaleFromPathname, missingTranslationMessage } from "~/i18n";
+
```

Comment:
BTW, edge case but this route doesn't catch the scenario of trying to go to the landing page (`index.mdx`) of a non translated language

Reply:
checking this! Thanks!